code/modules/mob/mob_helpers.dm

```diff
@@ -504,11 +504,13 @@
 		return FALSE
 
 ///Is the mob a flying mob
-/mob/proc/is_flying(mob/M = src)

```

Comment:
nothing used the param like this because why the hell would you lol

Reply:
:smirk: